src/lib/schematics/tree/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.ts

```diff
@@ -0,0 +1,113 @@
+import { Component<% if(!!viewEncapsulation) { %>, ViewEncapsulation<% }%><% if(changeDetection !== 'Default') { %>, ChangeDetectionStrategy<% }%> } from '@angular/core';
```

Comment:
Its used in every component schematic

Reply:
I commented on this in the other schematics too; I don't see a the need for it, and "because it was already like that" isn't a great reason for keeping it